armi/reactor/converters/axialExpansionChanger/expansionData.py

```diff
@@ -341,3 +355,14 @@ def isTargetComponent(self, c: "Component") -> bool:
             Component to check target component status
         """
         return bool(c in self._componentDeterminesBlockHeight)
+
+    def getTargetComponent(self, b: "Block"):
+        """Returns the target component for a block.
+
+        Parameters
+        ----------
+        b
+            the block to query for the target component
+        """
+        c = filter(lambda c: self.isTargetComponent(c), iterSolidComponents(b))
+        return next(c)
```

Comment:
this method is new in this PR. the rest of the edits in this file are from #2162